server/eclipse-project/src/main/java/us/freeandfair/corla/controller/ComparisonAuditController.java

```diff
@@ -137,8 +140,19 @@ private ComparisonAuditController() {
                               round, round.ballotSequence().size(),
                               round.ballotSequence()));
 
+    // we can't filter on "auditFlag" because we need to fetch these for the
+    // final review page.
+    Set<Long> previouslyAudited = rounds.stream()
+      .map(r -> r.ballotSequence())
+      .flatMap(List::stream)
+      .collect(Collectors.toSet());
+    List<Long> cvrIds = round.ballotSequence().stream()
+      .filter(cvrId -> !previouslyAudited.contains(cvrId))
```

Comment:
Does this end up filtering out ballots that were also audited in the current round? I can't tell but it seems that way reading the code. It would make sense to me if `rounds` were only rounds prior to the current round, rather than all rounds... but maybe there's a reason this works?

Reply:
I'm just testing it now, and yes, that first commit didn't work, but now it does.

Reply:
I see - the addition of line 146? Yep, that makes sense to me :+1: